heroku/cli
# refactor(pg-v5): Move command pg:promote to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bDLC6YAO/view

This converts `pg:promote` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Have an app ready with two databases. One database should already be set as `DATABASE_URL` (run `heroku pg:info --app APP to find out which one). DATABASE2 we'll use as the test to "promote" to be set as `DATABASE_URL`
3) Run `./bin/run pg:promote --app APP DATABASE2`
4) Verify the command succeeds and DATABASE 2 is set as `DATABASE_URL`
5) Compare with prod, reversing `DATABASE_URL` back to DATABASE1
